src/app/apps/rup/mapa-camas/views/censos/censo-mensual/censo-mensual.component.ts

```diff
@@ -191,6 +191,8 @@ export class CensosMensualesComponent implements OnInit {
 
         const totalEgreso = this.totales.egresosAlta + this.totales.egresosDefuncion;
         this.datosCensoTotal['promDiasEstada'] = (totalEgreso === 0) ? '0' : (this.totales.diasEstada / totalEgreso).toFixed(2);
+
+        this.datosCensoTotal['porcentajeCamasOcupadas'] = (this.censo.length > 0) ? ((this.totales.pacientesDia / this.totales.disponibles24) * 100).toFixed(2) : '0';
```

Comment:
No hace falta especificar > 0 , `this.censo.length` ya evalúa el mayor a 0

Reply:
Corregido!